larq_compute_engine/tests/end2end_test.py

```diff
@@ -152,43 +130,67 @@ def toy_model_int8(**kwargs):
 
 
 def preprocess(data):
-    return lqz.preprocess_input(data["image"])
+    return lqz.preprocess_input(data["image"]), data["label"]
 
 
-def assert_model_output(model_lce, inputs, outputs):
+def assert_model_output(model_lce, inputs, outputs, rtol, atol):
     interpreter = Interpreter(model_lce, num_threads=min(os.cpu_count(), 4))
     actual_outputs = interpreter.predict(inputs)
-    np.testing.assert_allclose(actual_outputs, outputs, rtol=0.001, atol=0.25)
+    np.testing.assert_allclose(actual_outputs, outputs, rtol=rtol, atol=atol)
 
 
 @pytest.mark.parametrize(
     "model_cls",
     [toy_model, toy_model_sequential, toy_model_int8, lqz.sota.QuickNet],
 )
 def test_simple_model(model_cls):
-    model = model_cls(weights="imagenet")
-    model_lce = convert_keras_model(
-        model, experimental_enable_bitpacked_activations=True
-    )
-
-    # Test on the flowers dataset
+    # Test on the OxfordFlowers dataset
```

Comment:
This is not the same as oxford flowers, see: https://www.tensorflow.org/datasets/catalog/tf_flowers
```suggestion
    # Test on the TF flowers dataset
```